src/config.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * Module for getting and setting Prebid configuration.
+ *
+ * Prebid previously defined these properties directly on the global object:
+ * pbjs.logging = true;
+ *
+ * Defining and access properties in this way is now deprecated, but these will
+ * continue to work during a deprecation window.
+ */
+const utils = require('./utils');
+
+const DEFAULT_DEBUG = false;
+const DEFAULT_BIDDER_TIMEOUT = 3000;
+const DEFAULT_PUBLISHER_DOMAIN = window.location.origin;
+const DEFAULT_COOKIESYNC_DELAY = 100;
+const DEFAULT_ENABLE_SEND_ALL_BIDS = false;
+
+const ALL_TOPICS = '*';
+
+let listeners = [];
```

Comment:
In this file:

```
export function newConfig() {
  let listeners = [];
  return {
    getConfig() { ... }
    setConfig(options) { ... }
    subscribe(topic, listener) { ... }
  };
}
```

In `prebid.js`:

```
import { newConfig } from './config';

const config = newConfig();
$$PREBID_GLOBAL$$.getConfig = config.getConfig
$$PREBID_GLOBAL$$.setConfig = config.setConfig
```

In modules:

```
const getConfig = $$PREBID_GLOBAL$$.getConfig
```

Unit tests don't need to worry about side-effects because they can call `newConfig()` each time.